katello_certs_tools/sslToolCli.py

```diff
@@ -241,11 +237,11 @@ def unique(s):
     if not n:
         return []
 
-    l = []
+    append_list = []
     for item in s:
-        if item not in l:
-            l.append(item)
-    return l
+        if item not in append_list:
+            append_list.append(item)
+    return append_list
```

Comment:
`return set(s)`? or what is this loop doing otherwise besides removing duplicates?